app/static/js/base.js

```diff
@@ -123,3 +123,9 @@ function shareOnReddit(url) {
 function getRandomUnsplash() {
 
 }
+
+$("#challenge-level").change(function(){ 
+  /* Be careful with SQL injections */
```

Comment:
I know I'm no longer really part of this project, but this comment just made me very interested :D. What is this referring to/where is the SQL injection to be careful of?